app/models/vm_or_template/operations.rb

```diff
@@ -112,11 +112,10 @@ def validate_vm_control
             end
       unsupported_reason_add(:control, msg) if msg
     end
-  end
 
-  def validate_terminate
-    return {:available => false, :message => 'The VM is terminated'} if self.terminated?
-    {:available => true, :message => nil}
+    supports :terminate do
```

Comment:
Yes, this is the only place. 
Miq UI doesn't have "terminate" operation.  It does have ```retire``` operation for VM.  However, by nature vm/template should be "terminable" as e.g. vSphere client have "Delete from disk" operation for both VM/templates. And that makes sense.

Also this change follow the original context/spirit which I think still makes sense.